client/state/data-layer/wpcom/sites/activity/test/index.js

```diff
@@ -80,13 +77,12 @@ describe( 'receiveActivityLog', () => {
 describe( 'receiveActivityLogError', () => {
 	it( 'should dispatch activity log error action', () => {
 		const dispatch = sinon.spy();
-		receiveActivityLogError( { dispatch }, { siteId: SITE_ID }, ERROR_RESPONSE );
-		expect( dispatch ).to.have.been.calledWith(
-			activityLogError( SITE_ID, {
-				error: 'unknown_blog',
-				message: 'Unknown blog',
-			} )
-		);
+		receiveActivityLogError( { dispatch } );
+		expect( dispatch ).to.have.been.called.once;
+		const errorNoticeDispatched = errorNotice( translate( 'Error receiving activity for site.' ) );
+		// Update noticeId because a second call will up it and it won't match for test.
+		errorNoticeDispatched.notice.noticeId = '1';
```

Comment:
more awkward…well I applaud the direction this is going.

it's too bad `errorNotice()` is so complicated here. maybe it would make sense to just test the basics and let the test be a little lax…

```js
expect( dispatch ).to.have.been.calledWithMatch( {
	type: errorNotice( 'blarg' ).type
} )
```

what do you think of that? what's the point of the test? to make sure we send an error notice?

Reply:
Yeah, that `receiveActivityLogError` creates an error notice.

Reply:
Updated, thanks for the review.